string name   #Name of path
float32 distance_thresh
---
bool pass